package.json

```diff
@@ -34,6 +35,7 @@
     "js-beautify": "^1.6.2",
     "raw-loader": "^0.5.1",
     "webpack": "^1.12.12",
-    "webpack-dev-server": "^1.14.1"
+    "webpack-dev-server": "^1.14.1",
+    "jsdoc": "^3.4.0"
```

Comment:
I'd bet you added this line manually instead of doing `npm install --save-dev jsdoc`. `npm install` sorts dependencies alphabetically, and having this line "unsorted" will generate noise in later commits (see https://github.com/iTowns/itowns2/pull/100/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2 for an example).

Could you sort the entries? (either manually, or remove the entry and let `npm install --save-dev` re-add it)